src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -64,6 +66,23 @@ def get_avatar_type(self) -> str:
     def class_name(self) -> str:
         return "User"
 
+    def get_option(
```

Comment:
This is interesting, and also, tricky.  A caller to `api_user.get_option` may not know how the user was loaded -- if the user was loaded with `options` on the original user, a call to `get_option` may return `None` or even the `default` value *although a real value exists, but was merely not loaded*.  I think that can be problematic and introduce subtle bugs quickly.

I have an idea of how to make the api slightly more onerous, but more explicit and less prone to this issue, if you don't mind.